### PR TITLE
Minor changes to snippets + p5js metadata

### PR DESCRIPTION
**Pull request by anthonygalea**
Encountered a couple of issues while on quil.info:
- without `no-loop` there are too many images saved in [save](http://quil.info/sketches/show/snippet_save) snippet
- `texture.jpg` is not loaded in [texture](http://quil.info/sketches/show/snippet_texture) snippet

### CODE REVIEW

File: src/cljc/quil/snippets/output.cljc [hunk 1]
```diff
@@ -18,14 +18,12 @@
...
+  {:renderer :p3d
+   :settings q/no-loop}
 
   (q/camera 150 150 150 0 0 0 0 0 1)
   (q/box 100)
   (q/save "generated/box.png")
-  (comment "stop sketch after saving image")
-  (comment "otherwise it will show save dialog")
-  (comment "on every iteration")
   (q/exit))
```
Comment (nbeloglazov) [inline src/cljc/quil/snippets/output.cljc]: Wny doesn't q/exit help here? It supposed to solve the same problem as q/no-loop.

Comment (anthonygalea) [reply src/cljc/quil/snippets/output.cljc]: Oh right, how did I miss that? Yes `q/exit` calls p5js `remove` which works here: https://editor.p5js.org/anthonygalea/sketches/pnmR05vm4

Comment (anthonygalea) [reply src/cljc/quil/snippets/output.cljc]: Oh, `q/exit` is not there when the sketch is opened in the editor:
Screenshot

Comment (anthonygalea) [reply src/cljc/quil/snippets/output.cljc]: Ah ok got it. The call to `exit` was simply added after snippets data generation for quil site. Will remove the commit containing the `no-loop` in this pull request and submit another pull request to update snippets data in quil-site.

**Comment by anthonygalea** [comment]
Sorry didn't realize this pull request is still open. It now also contains metadata for p5js.

**Comment by nbeloglazov** [comment]
We have a test that verifies that all processing links return 200: https://github.com/quil/quil/blob/master/test/clj/fn_metas.clj But looking at how p5js links look - I feel like we won't be able to write similar test?

**Comment by nbeloglazov** [comment]
Actually looking into p5js - they load this file: https://p5js.org/reference/data.min.json which I assume is used to render reference on client-side. We could use that file to check that all our p5js-names match. What do you think?

**Pull request merged**